source/fundamentals/crud/read-operations/cursor.txt

```diff
@@ -31,9 +31,8 @@ data from a `FindIterable
    other iterables such as an `AggregateIterable
    <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/AggregateIterable.html>`__.
 
-The ``find()`` method iterates through all the documents in your
-collection to find documents that match your query and returns an
-instance of a ``FindIterable``. 
+The ``find()`` method creates and returns an instance of a
+``FindIterable`` that contains documents matching your query. 
```

Comment:
nit: I know that Cho has nitpicked me before when I used "contains" in reference to an iterable, since cursors/iterables generally don't actually *contain* documents, they just point to them. Maybe this could be better worded:

```suggestion
``FindIterable`` that you can use to browse documents that match your query. 
```
... or something similar that avoids the use of "contains". (bonus points if you can get rid of the gerund, but that probably doesn't matter at all)

Reply:
Agreed, he has definitely nit picked that for me too. But after reading this sentence going into the next, I see that I'm repeating myself (the next sentence defines the iterable), so I'll be removing all that is after FindIterable here and editing the definition in the next sentence to not say "contains".